GVFS/GVFS.Mount/InProcessMount.cs

```diff
@@ -340,6 +341,11 @@ private void HandleReleaseLockRequest(string messageBody, NamedPipeServer.Connec
             }
 
             NamedPipeMessages.ReleaseLock.Response response = this.fileSystemCallbacks.TryReleaseExternalLock(request.RequestData.PID);
+            if (response.Result == NamedPipeMessages.ReleaseLock.SuccessResult)
+            {
+                this.tracer.SetGitCommandSessionId(string.Empty);

```

Comment:
Do we also need to call `SetGitCommandSessionId(string.Empty)` in `GVFSLock.ReleaseLockForTerminatedProcess`?

Reply:
(or in `ReleaseExternalLock`)?